packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -354,6 +355,11 @@ class HtmlElementView extends StatelessWidget {
   /// A PlatformViewFactory for this type must have been registered.
   final String viewType;
 
+  /// Callback to invoke after the platform view has been created.
+  ///
+  /// May be null.
```

Comment:
```suggestion
  /// {@macro flutter.widgets.AndroidView.onPlatformViewCreated}
```

Looks like we already have this exact doc in a macro